packages/design-system/src/images/close-alt.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="15" height="15" viewBox="0 0 15 15">
```

Comment:
Should this image name also have the `usa-banner` prefix?

Reply:
I wasn't planning to as it's another version of the `close` svg and can probably be used by any other components.